cpp-client/README.md

```diff
@@ -6,7 +6,9 @@ installation, into /usr/local, assuming C++17. We intend to provide instructions
 variations in the future.
 
 1. Start with an Ubuntu 20.04 install
+
 2. Get Deephaven running by following the instructions here: https://deephaven.io/core/docs/how-to-guides/launch-build/
```

Comment:
I think we should tell them to build from source for now, because things change a lot form release to release so it is hard to ensure stuff continues to work with the old release as we change things.  This way we guarantee that stuff works with latest that is likely what we tested against.